perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -93,25 +106,157 @@ def Prepare(benchmark_spec):
   vms = vms[:2]
   vm_util.RunThreaded(PrepareNetperf, vms)
 
+  num_streams = FLAGS.netperf_num_streams
+
+  # Start the netserver processes
   if vm_util.ShouldRunOnExternalIpAddress():
-    vms[1].AllowPort(COMMAND_PORT)
-    vms[1].AllowPort(DATA_PORT)
+    # Open all of the command and data ports
+    vms[1].AllowPort(PORT_START, PORT_START + num_streams * 2 - 1)
+  netserver_cmd = ('for i in $(seq {port_start} 2 {port_end}); do '
+                   '{netserver_path} -p $i & done').format(
+                       port_start=PORT_START,
+                       port_end=PORT_START + num_streams * 2 - 1,
+                       netserver_path=netperf.NETSERVER_PATH)
+  vms[1].RemoteCommand(netserver_cmd)
+  # vms[1].RemoteCommand('%s -p %s' % (netperf.NETSERVER_PATH, COMMAND_PORT))
+
+  # Install some stuff on the client vm
+  vms[0].Install('pip')
+  vms[0].RemoteCommand('sudo pip install python-gflags==2.0')
+
+  # Copy remote test script to client
+  scratch_dir = vms[0].GetScratchDir()
+  vms[0].RemoteCommand('sudo mkdir -p %s/run/' % scratch_dir)
+  vms[0].RemoteCommand('sudo chmod 777 %s/run/' % scratch_dir)
+  for file_name in REMOTE_SCRIPT_FILES:
```

Comment:
I kept it for flexibility, yeah. It would be good to have some first-class support for remote scripts since we're using them in multiple benchmarks now. I opted to just remove the unnecessary logic for now, though.